src/test/java/org/opentest4j/AssertionFailedErrorTests.java

```diff
@@ -158,6 +158,23 @@ public void deserializationOfAssertionFailedErrorWorksForVersion_1_0_0() throws
 		assertEquals("bar", error.getActual().getValue());
 	}
 
+	@Test
+	public void ephemeralValueIsOmittedFromSerialization() throws Exception {
```

Comment:
Hm, wouldn't this make more sense if `value` itself was `Serializable`, maybe add another variant with that?

Reply:
The original intent was to verify the behavior of `ephemeralValue` through serialization, which should return null when given a non-serializable value.

I'm not opposed to adding more test but there are already a handful that exercise `AssertionFailedError` and `ValueWrapper` with serializable values.